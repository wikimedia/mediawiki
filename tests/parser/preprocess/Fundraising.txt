<div name="fundraising" id="fundraising" class="plainlinks" style="margin-top:5px; text-align: center; background-color: #ffffe0; border: solid 1px #e0e0c0">
'''Pwede kang [[Wikimedia:give the gift of knowledge|maghandog ng kaalaman]] sa paraan ng [[Wikimedia:Fundraising#Donation_methods|pagbibigay ng donasyon sa Pundasyong Wikimedia!]]'''
<br />
<fundraising/>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<fundraisinglogo/>
<br />
<b>Ngayon, ang iyong [[Wikimedia:Fundraising|kontribusyon]] ay [[Wikimedia:Fundraising FAQ|itatambal]] ng isang anonimong kaibigan.</b>
<br />
<small>
[[Wikimedia:Deductibility of donations|Pagbabawas sa mga buwis ng donasyon]]
|
[[Wikimedia:Fundraising FAQ|FAQ]]
|
[http://upload.wikimedia.org/wikipedia/foundation/2/28/Wikimedia_2006_fs.pdf Mga pampananalaping pahayag]
</small>
</div>
