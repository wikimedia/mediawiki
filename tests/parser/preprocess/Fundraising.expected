<root>&lt;div name=&quot;fundraising&quot; id=&quot;fundraising&quot; class=&quot;plainlinks&quot; style=&quot;margin-top:5px; text-align: center; background-color: #ffffe0; border: solid 1px #e0e0c0&quot;&gt;
'''Pwede kang [[Wikimedia:give the gift of knowledge|maghandog ng kaalaman]] sa paraan ng [[Wikimedia:Fundraising#Donation_methods|pagbibigay ng donasyon sa Pundasyong Wikimedia!]]'''
&lt;br /&gt;
&lt;fundraising/&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;
&lt;fundraisinglogo/&gt;
&lt;br /&gt;
&lt;b&gt;Ngayon, ang iyong [[Wikimedia:Fundraising|kontribusyon]] ay [[Wikimedia:Fundraising FAQ|itatambal]] ng isang anonimong kaibigan.&lt;/b&gt;
&lt;br /&gt;
&lt;small&gt;
[[Wikimedia:Deductibility of donations|Pagbabawas sa mga buwis ng donasyon]]
|
[[Wikimedia:Fundraising FAQ|FAQ]]
|
[http://upload.wikimedia.org/wikipedia/foundation/2/28/Wikimedia_2006_fs.pdf Mga pampananalaping pahayag]
&lt;/small&gt;
&lt;/div&gt;
</root>